export const task = new Promise((resolve, reject) => {
    resolve(true);
    //reject("Message error from Reject")
});
